doc/user_guide/Server_Deployment.md

```diff
@@ -209,6 +209,47 @@ Finally, to deploy a Panel app to App Engine run `gcloud app create` and `gcloud
 
 For detailed information and steps, check out this [example](https://towardsdatascience.com/deploy-a-python-visualization-panel-app-to-google-cloud-cafe558fe787?sk=98a75bd79e98cba241cc6711e6fc5be5) on how to deploy a Panel app to App Engine and this [example](https://towardsdatascience.com/deploy-a-python-visualization-panel-app-to-google-cloud-ii-416e487b44eb?sk=aac35055957ba95641a6947bbb436410) on how to deploy a Panel app to Cloud Run.
 
+#### Hugging Face
+
+The guides below assumes you have already signed up and logged into your account at [huggingface.co](https://huggingface.co/).
+
+#### Duplicate an existing space
+
+The easiest way to get started is to  [search](https://huggingface.co/spaces), find and duplicate an existing space. A simple space is
+[MarcSkovMadsen/awesome-panel](https://huggingface.co/spaces/MarcSkovMadsen/awesome-panel).
+
+- Open the space [MarcSkovMadsen/awesome-panel](https://huggingface.co/spaces/MarcSkovMadsen/awesome-panel).
+- Click the 3 dots and select *Duplicate this Space*.
+
+<img src="../_static/hugging-face-duplicate.png" style="width:67%"></img>
+
+- Follow the instructions to finish the duplication.
+
+Once you have finalized the duplication you will need to take a look at the `app.py` file in the new space to figure out what to replace.
+
+In this example you will need to
+
+- Update the contents of the `panel_app.py` file.
```

Comment:
@philippjfr . Would you also suggest using `subprocess` to run `panel serve some_other_file.py`. Or would you recommend we suggest using `pn.serve`?

Reply:
I see no issue with running `pn.serve`.

Reply:
The issue I see is that the `api` is different. You can't use for example the `.servable()` api in the same way. And users often stumble because they instantiate shared instances.